libraries/stdlib/common/src/generated/_Ranges.kt

```diff
@@ -961,6 +961,7 @@ public fun CharProgression.reversed(): CharProgression {
 
 /**
  * Returns a progression that goes over the same range with the given step.
+ * @sample samples.ranges.Ranges.intStep
```

Comment:
This file is generated. Please change the generator code in `libraries/tools/kotlin-stdlib-gen/src/templates/Ranges.kt` (`f_step` function) and run "Generate standard library sources" run configuration in IDEA, otherwise these changes will be lost.

Reply:
> This file is generated. Please change the generator code in `libraries/tools/kotlin-stdlib-gen/src/templates/Ranges.kt` (`f_step` function) and run "Generate standard library sources" run configuration in IDEA, otherwise these changes will be lost.

done